lib/src/rules/prefer_initializing_formals.dart

```diff
@@ -35,6 +35,41 @@ class Point {
 }
 ```
 
+**BAD:**
+```
+class Point {
+  num x, y;
+  Point({num x, num y}) {
+    this.x = x;
+    this.y = y;
+  }
+}
+```
+
+**GOOD:**
+```
+class Point {
+  num x, y;
+  Point({this.x, this.y});
+}
+```
+
+**NOTE**
+Named parameters must match with the field name in order to be consider by the
+lint to avoid having to update either the field or the named argument and to
+allow for the API to accomodate for different guidelines for named arguments
+and fields currently proposed in https://dart.dev/guides/language/effective-dart
+thus the following example won't trigger the lint:
+
```

Comment:
/cc @bwilkerson for word-smithing ideas.

(After 3 tries, it's time for a life-line!)

Reply:
The biggest improvement would be to split this up into multiple sentences. Consider something like the following:

This rule will not generate a lint for named parameters unless the parameter name and the field name are the same. The reason for this is that resolving such a lint would require either renaming the field or renaming the parameter, and both of those actions would potentially be a breaking change. For example, the following will not generate a lint:

Reply:
Excellent.  Thanks @bwilkerson!

@dramos07: does that sound OK to you?

Reply:
That was definitely an improvement thanks @bwilkerson and @pq